src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -81,8 +84,8 @@ public DefaultClient(URL baseUrl,  OkHttpClient client, IResourceFactory factory
 		if(this.factory != null) {
 			this.factory.setClient(this);
 		}
-		openShiftVersion = System.getProperty(SYSTEM_PROP_OPENSHIFT_API_VERSION, null);
-		kubernetesVersion = System.getProperty(SYSTEM_PROP_K8E_API_VERSION, null);
+		initMasterVersion("version/openshift", new VersionCallback(version -> this.openShiftVersion = version));
```

Comment:
@jcantrill `openShiftVersion` and `kubernetesVersion` are used nowhere, except `DafaultClient`. Moreover, it doesn't have sense to be these variables responsible for API version, because it doesn't really provide any useful information. It is always v1 (agree, that some long time ago it was v1beta). Plus, if it is API versions why are 2 variables used then? Anyway the versions are the same - look at `OpenShiftAPIVersion` and `KubernetesAPIVersion` enums - they're exact copies of each others